docs/user-guide/model-inputs.md

```diff
@@ -1,6 +1,8 @@
 # Model Inputs
 
 These are the input files needed to run SIPNET:
+
+See [Input Files](../parameters.md#input-files) for the parameter definitions and detailed section anchor.
```

Comment:
This is a circular link! Perhaps "Input Files" and "Outputs" sections should be removed from the parameters.md? 


```suggestion
```


from https://pecanproject.github.io/sipnet/parameters/#input-files:

<img width="412" height="207" alt="Image" src="https://github.com/user-attachments/assets/bd15b2cc-863b-48f7-9744-295278b15baa" />